### ISSUE TITLE
Docker doesn't update to last version

### ISSUE THREAD

**Issue by simonbcn** [sbondCo/Watcharr#491]
```
Arch Linux
Docker version 26.0.1, build d260a54c81
```
```
$ cat docker-compose.yaml
services:
  watcharr:
    # The :latest tag is used for simplicity, it is recommended
    # to use an actual version, then when updating check the releases for changelogs.
    image: ghcr.io/sbondco/watcharr:latest
    container_name: watcharr
#    network_mode: "host"
    #restart: always
    user: 1000:1000
    ports:
      - 3080:3080
    volumes:
      # Contains all of watcharr data (database & cache)
      - ./data:/data
```

I update with `docker compose pull && docker compose down && docker compose up -d`:
```
[+] Pulling 1/1
✔ watcharr Pulled        
[+] Running 2/2
✔ Container watcharr        Removed
✔ Network watcharr_default  Removed
[+] Running 2/2
✔ Network watcharr_default  Created   
✔ Container watcharr        Started 
```

But the version is still `1.37.0`.

**Comment by IRHM** [comment]
Thanks for reporting this! You're right, the `latest` tag is stuck on the 1.37 release for some reason.

I recently changed the release workflow to support publishing to the docker hub, looks like i broke something in the process, will have a look shortly.

For now if you want to update, you can swap `latest` with `v1.38.1`

**Comment by IRHM** [comment]
Fixed in [v1.38.2](https://github.com/sbondCo/Watcharr/releases/tag/v1.38.2). Thanks again for reporting this issue, please let me know if you have problems with anything else.
